nemo/collections/asr/models/__init__.py

```diff
@@ -12,11 +12,27 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from nemo.collections.asr.models.asr_model import ASRModel
-from nemo.collections.asr.models.classification_models import EncDecClassificationModel
-from nemo.collections.asr.models.clustering_diarizer import ClusteringDiarizer
-from nemo.collections.asr.models.ctc_bpe_models import EncDecCTCModelBPE
-from nemo.collections.asr.models.ctc_models import EncDecCTCModel
-from nemo.collections.asr.models.label_models import EncDecSpeakerLabelModel, ExtractSpeakerEmbeddingsModel
-from nemo.collections.asr.models.rnnt_bpe_models import EncDecRNNTBPEModel
-from nemo.collections.asr.models.rnnt_models import EncDecRNNTModel
+# TODO @blisc: Perhaps refactor instead of import guarding
+try:
```

Comment:
Why is all of this required? I dont want this kind of guard over all domains, it makes NeMo un-maintanable, having to put shim classes in the except. This comment applies to every single place where such import guard is placed with shim in except case. Ordinary import guards are fine but not this shim style.

Reply:
Original prototype showed none of this was required and the torch tts block could be imported with no modifications to imports in other domains.

Reply:
Because tts depends on code from asr. So there are a few options:
1) What is done in the above file
2) Copy and pasting code and deleting all files that reference lightning, hydra, omegaconf
3) Delete all of the offending imports from __init__.py. IE just have empty __init__.py's.

If you can think of anything else, let me know. IMO, all options are bad in their own way.

Reply:
Agreed, lets refactor in a future PR